lib/dora.ex

```diff
@@ -65,12 +65,12 @@ defmodule Dora do
     :ets.insert(:address_instances, {address, pid})
   end
 
-  def store_contract_information(address, timestamp, abi_path) do
-    :dets.insert(:addresses, {address, timestamp, abi_path})
+  def store_contract_information(address, block, abi_path) do
+    :dets.insert(:addresses, {address, block, abi_path})
 
     Dora.Contracts.create_or_update_contract(address, %{
       address: address,
-      last_timestamp: timestamp,
+      last_block: block,
```

Comment:
Just double checking that you are not actually storing the entire block here and it's just the block number, right?

Reply:
Nop, it's just the number. The function that calls this one does:

```
last_block =
  List.last(events)["blockNumber"]
  |> Utils.hex_to_int()
```

It just gets the number from the latest Event obtained.

Reply:
That's what I figured. At first glance, was just confused by the name of the parameter being `block`

Reply:
I can change it to `block_number`. What do you think?

Reply:
Might be best, actually. Just to increase readability of the function

Reply:
Done. Check it now